CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing to the Self Hosted Show Wiki
+
+
+## Objectives of This Wiki
+
+In order to contribute well, you need to understand what the objectives are behind this wiki.
+
+What we are trying to create:
+- A compilation of tools, tips and tricks for self-hosting specific applications.
+- A place where beginners and experts can learn about how to quickly, efficiently and securely deploy applications on their own server(s).
+- Snippets that people can quickly reference in deploying applications.
+- A hub for quality documentation (to the extent possible).
+
+What we are not trying to do:
+- Be another large list of self-hosted tools. See [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted) for a comprehensive list of self-hosted applications.
+- Be a hub for script kitties. People should _learn_ from documentation here not just come here to copy/paste. Snippets are important but should be documented properly.
+- Insecure, sloppy or any undesirable systems administration practices such as:
```

Comment:
This should flow slightly nicer as part of the other list items:

```suggestion
- Promote or encourage Insecure, sloppy or any undesirable systems administration practices such as:
```

Reply:
Added this suggestion in a38193235

Reply:
@RealOrangeOne in regard to the stray newlines in the doc, do you have suggestions on how you would organize it? I tried to reduce newlines but would love to see how you would organize it.

Reply:
> in regard to the stray newlines in the doc

Re-reading it, ignore me, I think what's there is fine. Not sure what I was talking about when I commented!